### PR TITLE
Add token counts to compute performance

### PR DESCRIPTION
**Pull request by rasbt**
This is an update to the finetuning scripts to report the tok/sec performance. Right now, it's based on the total tokens with padding (that's how other tools report it), but the raw token counts are added as well to compute it differently based on the project's needs. Examples:

### Batch size 1

```
| ------------------------------------------------------
| Token Counts
| - Input Tokens              :   1578
| - Tokens w/ Prompt          :   2674
| - Total Tokens (w/ Padding) :   2674
| -----------------------------------------------------
| Performance
| - Training Time             :  43.95 s
| - Tok/sec                   :  60.85 tok/s
| -----------------------------------------------------
| Memory Usage                                                                 
| - Memory Used               :  10.45 GB                                        
=======================================================
```

### Batch size 2

```
| ------------------------------------------------------
| Token Counts
| - Input Tokens              :   1578
| - Tokens w/ Prompt          :   2674
| - Total Tokens (w/ Padding) :   3348
| -----------------------------------------------------
| Performance
| - Training Time             :  30.27 s
| - Tok/sec                   :  110.61 tok/s
| -----------------------------------------------------
| Memory Usage                                                                 
| - Memory Used               :  10.66 GB                                        
=======================================================
```

### CODE REVIEW

File: litgpt/finetune/lora.py [hunk 1]
```diff
@@ -105,6 +105,7 @@ def setup(
     checkpoint_dir = auto_download_checkpoint(model_name=checkpoint_dir, access_token=access_token)
     pprint(locals())
     data = Alpaca() if data is None else data
+
```
Comment (rasbt) [inline litgpt/finetune/lora.py:108]:
```suggestion
```
